memex/src/test/java/com/johnlpage/memex/cucumber/steps/InspectionSteps.java

```
@@ -47,18 +62,70 @@ public Boolean bool(String bool){
     }
 
     @Given("the following vehicle inspections exist:")
-    public void givenVehicleInspectionsExist(DataTable dataTable) {
-        // test assumes it exists however it would be better to create it here
+    public void givenVehicleInspectionsExist(DataTable dataTable) throws JsonProcessingException {
+        List<Map<String, String>> rows = dataTable.asMaps(String.class, String.class);
+        List<VehicleInspection> inspections = new ArrayList<>();
+
+        for (Map<String, String> row : rows) {
+            String json = row.get("vehicleInspection");
+            VehicleInspection inspection = objectMapper.readValue(json, VehicleInspection.class);
+            inspections.add(inspection);
+        }
+
+        List<VehicleInspection> filtered = inspections.stream()
```

I don't think we should quietly filter - if the range boundary is violated we need to throw an exception